routes/api_v1_0.py

```diff
@@ -1476,13 +1474,13 @@ def _is_cache_valid(dataset: str, f: str) -> bool:
     """
 
     config = DatasetConfig(dataset)
-    if os.path.isfile(f):
+    if Path(f).is_file():
         cache_time = config.cache
         if cache_time is not None:
-            modtime = datetime.datetime.fromtimestamp(os.path.getmtime(f))
+            modtime = datetime.datetime.fromtimestamp(f.stat().st_mtime)
```

Comment:
Is `f` a Path object here? I ask because you use `Path(f)` on 1477 and 1483. If not, create one Path object from `f` above and use it accordingly.

Reply:
Hi Justin, f is a poxis path, called from tiles route. I changed it accordingly.
Path(f).is_file(): to f.is_file():

and also f.unlink() in line 1483